src/lib/marshal/email_layout/helpers.ts

```diff
@@ -26,3 +26,19 @@ export const formatFooterLinks = (
       .join(", ") + ` (+ ${count - limit} more)`
   );
 };
+
+export const getFooterLinksList = (
+  emailLayout: EmailLayoutData,
+):
+  | {
+      key: string;
+      value: string;
+    }[]
+  | [] => {
+  const { footer_links } = emailLayout;
+  if (!footer_links) return [];
+
+  return footer_links.map((link) => {
+    return { key: link.text, value: link.url };
```

Comment:
This function returning a list of `{ key: ..., value: ... }` feels pretty specific to how oclif ux table function works - so I would probably just do it directly there?